### PR TITLE
Add PCL built schemas into unit tests

### PR DESCRIPTION
**Pull request by AndyDentFree**
Simple tests added on top of the earlier PCL fix to see if could trigger bug seen in `QuickJournal`. however, these work nicely.

ObjectIntegrationTests.cs
- CreateObjectFromPCLTest added

InstanceTests.cs
- GetInstanceFromPCLTest added

IntegrationTests.XamarinAndroid
IntegrationTests.XamarinIOS
- added reference to PurePCLBuildableTest

PurePCLBuildableTest/Class1.cs
- renamed to ObjectInPCL
- added properties of each type

PurePCLBuildableTest
- TestBuildingRealmFromPCL.MakeARealmWithPCL added

### CODE REVIEW

File: Tests/IntegrationTests.Shared/InstanceTests.cs [hunk 1]
```diff
@@ -62,6 +62,16 @@ public void GetInstanceWithJustFilenameTest()
             // Arrange, act and "assert" that no exception is thrown, using default location + unique name
             Realm.GetInstance(specialRealmName).Close();
         }
+        
+        [Test]
```
Comment (kristiandupont) [inline Tests/IntegrationTests.Shared/InstanceTests.cs]: Might we not just as well move this (and the `ObjectIntegrationTests.CreateObjectFromPCLTest` test below) into the PCL themselves? Seems to me that they belong in there.

File: Tests/PurePCLBuildableTest/ObjectInPCL.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+﻿using System;
```
Comment (kristiandupont) [inline Tests/PurePCLBuildableTest/ObjectInPCL.cs]: Copyright header missing

File: Tests/IntegrationTests.Shared/InstanceTests.cs (hunk 1 above)
Comment (AndyDentFree) [inline Tests/IntegrationTests.Shared/InstanceTests.cs]: Can we put the unit tests inside the PCL? I figured that opened a bigger can of worms plus I wanted to explicitly test calling functions **from** native code, that 

1. use a PCL object on a realm in the native app and
2. go through the whole Realm creation **in** the PCL

**Comment by kristiandupont** [comment]
:+1:

**Pull request merged**